examples/LLM/llama2/chat_app/model-config.yaml

```diff
@@ -0,0 +1,7 @@
+# TorchServe frontend parameters
+responseTimeout: 1200
+
+handler:
+    model_name: "llama-cpp"
+    model_path: "/Users/agunapal/Documents/experiments/llama/ggml-model-q4_0.gguf"
```

Comment:
update this path or error in handler if its not set

Reply:
I have taken care of it. I am setting an env variable and using that. But let me add a comment saying that users should updated this path in real usecase https://github.com/pytorch/serve/pull/2618/files#diff-f8d92d4ecdf7cd36b36149d88f699c651f5f1da2dc874ca13501bf524b1f6968R29